packages/components/src/Draggable/DraggableComponent.test.js

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import TestBackend from 'react-dnd-test-backend';
+import { DragDropContext as dndContext } from 'react-dnd';
+import PropTypes from 'prop-types';
+import TestUtils from 'react-dom/test-utils';
+import draggable from './DraggableComponent.js';
+
+function Element({ element }) {
+	return (
+		<div className="elem-class">
+			<span>{element.symbol}</span>
+			<span>:</span>
+			<span>{element.name}</span>
+		</div>
+	);
+}
+
+Element.propTypes = {
+	element: PropTypes.object,
+};
+
+function List({ elements, Comp, extra }) {
+	return (
+		<div className="list-class">
+			{elements.map(elem => <Comp key={elem.symbol} element={elem} extra={extra} />)}
+		</div>
+	);
+}
+
+List.propTypes = {
+	elements: PropTypes.array,
+	Comp: PropTypes.func,
+	extra: PropTypes.object,
+};
+
+const ru = {
+	symbol: 'ru',
+	name: 'Ruthenium',
+};
+
+const rh = {
+	symbol: 'rh',
+	name: 'Rhodium',
+};
+
+const pd = {
+	symbol: 'pd',
+	name: 'Palladium',
+};
+
+const elements = [ru, rh, pd];
```

Comment:
Please define those inline inside the array.

Reply:
But i use the ru and rh objects later in the test.

Reply:
OK.